### ISSUE TITLE
'Deprecated' causes FatalErrorException on dashboard

### ISSUE THREAD

**Issue by jadwigo** [bolt/bolt#7090]
What happens: An error is displayed and the CSS rules for the layout are broken.
Bolt version: Bolt 3.4.0 beta 3
URL: On the dashboard page
Where: In the system log area

How does it look:
![image](https://user-images.githubusercontent.com/1488214/31616121-1e2fb1b0-b28c-11e7-8994-6f1de290aa01.png)

The call to `use Bolt\Common\Deprecated;` causes and error in the system log in the
```
FatalErrorException in Str.php line 5: 
Compile Error: Cannot use Bolt\Common\Deprecated as Deprecated because the name is already in use
```
Removing the line makes the error message go away
https://github.com/bolt/bolt/blob/9ccb3a01de8f2fec97b2ac6ab98d26b9751a2397/src/Helpers/Str.php#L5

**Comment by GawainLynch** [comment]
What is being called?

**Comment by jadwigo** [comment]
If I comment line 5 out it shows:
![image](https://user-images.githubusercontent.com/1488214/31616916-497f9176-b28e-11e7-87bc-9b0be5a43de7.png)

**Comment by GawainLynch** [comment]
Yeah, sorry … typing "Quickly Confused" … what I meant was "what is the trace leading up to this?"

Reason I am asking is that to my knowledge core shouldn't be calling those deprecated functions, and I can't reproduce it here … so my concern is other lurking :koala:s

**Comment by jadwigo** [comment]
In the synfony profiler I see this:
```
Exceptions

Whoops, looks like something went wrong.

1/1
FatalErrorException in Str.php line 5:
Compile Error: Cannot use Bolt\Common\Deprecated as Deprecated because the name is already in use
in Str.php line 5
at DebugClassLoader->loadClass() in 342aa97f89bb671062818cfada34f7c3502411f8a9e3cb70d62850d8857e225f.php line 177
at spl_autoload_call() in 342aa97f89bb671062818cfada34f7c3502411f8a9e3cb70d62850d8857e225f.php line 177
at HtmlRuntime->shy() in 342aa97f89bb671062818cfada34f7c3502411f8a9e3cb70d62850d8857e225f.php line 152
at __TwigTemplate_931ca5979e2db25e184cfc062b866c7d1aa7e946405aadf9b1e6065266ea8003->block_panel_body() in Template.php line 215
at Twig_Template->displayBlock() in 2f5ace2a5306b0416d5d3e1cfb6a156f56f1bad0571cef85f8e09e7df0423027.php line 71
at __TwigTemplate_a74de0484ba72ed68e965123522ce015448cb693ae45f2f636a6415ec02f1ccc->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in 342aa97f89bb671062818cfada34f7c3502411f8a9e3cb70d62850d8857e225f.php line 53
at __TwigTemplate_931ca5979e2db25e184cfc062b866c7d1aa7e946405aadf9b1e6065266ea8003->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in Template.php line 411
at Twig_Template->render() in Core.php line 1491
at twig_include() in 40f75d7a18064baabc37407d86cffcf154b91ba8003f695cda69bf263ba428dd.php line 50
at __TwigTemplate_80e5c592ffd09dbcd7c17c6800ededa33950a0db6ef7a72a910a4715f1a466a2->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in Template.php line 411
at Twig_Template->render() in Environment.php line 363
at Twig_Environment->render() in TemplateViewListener.php line 59
at TemplateViewListener->render() in TemplateViewListener.php line 45
at TemplateViewListener->onView() in WrappedListener.php line 61
at call_user_func:{/mnt/raid/2kdata/dev/____users/lodewijk/proxy2k/vendor/symfony/event-dispatcher/Debug/WrappedListener.php:61}() in WrappedListener.php line 61
at WrappedListener->__invoke() in WrappedListener.php line 61
at call_user_func:{/mnt/raid/2kdata/dev/____users/lodewijk/proxy2k/vendor/symfony/event-dispatcher/Debug/WrappedListener.php:61}() in WrappedListener.php line 61
at WrappedListener->__invoke() in EventDispatcher.php line 184
at call_user_func:{/mnt/raid/2kdata/dev/____users/lodewijk/proxy2k/vendor/symfony/event-dispatcher/EventDispatcher.php:184}() in EventDispatcher.php line 184
at EventDispatcher->doDispatch() in EventDispatcher.php line 46
at EventDispatcher->dispatch() in TraceableEventDispatcher.php line 138
at TraceableEventDispatcher->dispatch() in HttpKernel.php line 147
at HttpKernel->handleRaw() in HttpKernel.php line 62
at HttpKernel->handle() in InlineFragmentRenderer.php line 83
at InlineFragmentRenderer->render() in FragmentHandler.php line 134
at FragmentHandler->render() in HttpKernelExtension.php line 60
at HttpKernelExtension->renderFragment() in e2bee3812ac65dd324d25e0c6aa58b82e5de47ba160bb1ea843167d8f3f4063f.php line 81
at __TwigTemplate_bc64adba26ee5e2520d0529ab5c14453db3266e426b45c95622b8adb35fe9037->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in Template.php line 411
at Twig_Template->render() in Core.php line 1491
at twig_include() in a29b3fae59435b6f40f7cfc1d743405b8e89a4a2628fe984c5e08286778c0084.php line 200
at __TwigTemplate_71535575b949a46d799bb8b739aa2a7a7386703144fdd2634c4c02de09e617bb->block_page_main() in Template.php line 215
at Twig_Template->displayBlock() in 91736ffa842c6ddfc7b0b3d47375a26e6cf1cb4856aced91a475c8dfc27f0830.php line 175
at __TwigTemplate_7d26ca929e988f0ac6e4a242b8d74ee47c93c8d71cef7463bb5fcf249d6706f1->block_page_plain() in Template.php line 215
at Twig_Template->displayBlock() in 2aa44946e0a7c5c091d988f546fb44d23d11cfb91332c6e12a2153828b2b4a3f.php line 202
at __TwigTemplate_601305d186ecd78be4e4b8e8688cb184991d6e6e7c4436486f674657a6518fb2->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in 91736ffa842c6ddfc7b0b3d47375a26e6cf1cb4856aced91a475c8dfc27f0830.php line 61
at __TwigTemplate_7d26ca929e988f0ac6e4a242b8d74ee47c93c8d71cef7463bb5fcf249d6706f1->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in a29b3fae59435b6f40f7cfc1d743405b8e89a4a2628fe984c5e08286778c0084.php line 54
at __TwigTemplate_71535575b949a46d799bb8b739aa2a7a7386703144fdd2634c4c02de09e617bb->doDisplay() in Template.php line 432
at Twig_Template->displayWithErrorHandling() in Template.php line 403
at Twig_Template->display() in Template.php line 411
at Twig_Template->render() in Environment.php line 363
at Twig_Environment->render() in TemplateViewListener.php line 59
at TemplateViewListener->render() in TemplateViewListener.php line 45
at TemplateViewListener->onView() in WrappedListener.php line 61
at call_user_func:{/mnt/raid/2kdata/dev/____users/lodewijk/proxy2k/vendor/symfony/event-dispatcher/Debug/WrappedListener.php:61}() in WrappedListener.php line 61
at WrappedListener->__invoke() in EventDispatcher.php line 184
at call_user_func:{/mnt/raid/2kdata/dev/____users/lodewijk/proxy2k/vendor/symfony/event-dispatcher/EventDispatcher.php:184}() in EventDispatcher.php line 184
at EventDispatcher->doDispatch() in EventDispatcher.php line 46
at EventDispatcher->dispatch() in TraceableEventDispatcher.php line 138
at TraceableEventDispatcher->dispatch() in HttpKernel.php line 147
at HttpKernel->handleRaw() in HttpKernel.php line 62
at HttpKernel->handle() in Application.php line 586
at Application->handle() in Application.php line 563
at Application->run() in Application.php line 97
at Application->run() in index.php line 12
at {main}() in index.php line 0
```

**Comment by jadwigo** [comment]
And:
![image](https://user-images.githubusercontent.com/1488214/31617497-a71ab2a6-b28f-11e7-9e25-440ffc36aed2.png)

**Comment by jadwigo** [comment]
My guess is that the system activity block is doing something oldfashioned

**Comment by jadwigo** [comment]
I tries to see if it works if I run `app/nut setup:sync` again .. that helped for a moment.

But when I run a cache clear the error is back

**Comment by jadwigo** [comment]
So apparently in `{{ log.message|raw|shy }}` the `|shy` part triggered the error message.

I've changed that to `|excerpt(32)` .. which should be usable as an indicator of what the system message was on the dashboard.

Maybe the soft hyphenate filter is using something deprecated.. but I'll leave that for someone else to find.

**Comment by GawainLynch** [comment]
I believe, from testing, it is [this PHP bug](https://bugs.php.net/bug.php?id=66773) (and [related](https://bugs.php.net/bug.php?id=71079)).

**Comment by bobdenotter** [comment]
Fixed in #7094
